samples/show-case/cloud-azure/sample.md

```diff
@@ -14,6 +14,23 @@ This sample implements a fictional store that can be deployed to Azure. It is di
 ## Feature usage
 
 
+### Azure Storage Queues Transport
+
+All endpoints in the solution communicate using the [Azure Storage Queues transport](/nservicebus/azure-storage-queues/index.md). The sample has been configured to use the Azure Storage Emulator. 
+
+
+### Azure Storage Persistence
+
+The endpoints in the solution persist data using the [Azure Storage persistence](/nservicebus/azure-storage-persistence/index.md). This persistence is used to store subscription, timeout, and saga data.
+
+NOTE: The Azure Storage persistence does not support collection types. The `ProcessOrderSaga` avoids this issue by combining a collection of `ProductIds` into a single string for persistence.
```

Comment:
Unfortunately it still throws an exception saying 

> The property type `String[]` is not supported in windows azure table storage

I did some investigation and it appears that the change in https://github.com/Particular/NServiceBus.Persistence.AzureStorage/pull/99 was [reverted](https://github.com/Particular/NServiceBus.Persistence.AzureStorage/commit/88d1b2b3792b1b179ecab92e76fb1e11f4da25bd) by @Scooletz

Perhaps he can provide some more context?